server/src/test/java/org/solarbank/server/integration/ServerApplicationIT.java

```diff
@@ -2,8 +2,10 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
+@ActiveProfiles("test")
```

Comment:
Runs the the tests in the `test` profile so as to pick up the SSL config from `application-test.yml`.